lemminx-maven/src/test/java/org/eclipse/lemminx/extensions/maven/participants/hover/ManagedVersionHoverTest.java

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Red Hat Inc. and others.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.lemminx.extensions.maven.participants.hover;
+
+import static org.eclipse.lemminx.extensions.maven.utils.MavenLemminxTestsUtils.createDOMDocument;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.concurrent.ExecutionException;
+
+import org.eclipse.lemminx.dom.DOMDocument;
+import org.eclipse.lemminx.extensions.maven.NoMavenCentralExtension;
+import org.eclipse.lemminx.services.XMLLanguageService;
+import org.eclipse.lemminx.settings.SharedSettings;
+import org.eclipse.lsp4j.Hover;
+import org.eclipse.lsp4j.MarkupContent;
+import org.eclipse.lsp4j.Position;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(NoMavenCentralExtension.class)
+public class ManagedVersionHoverTest {
+
+	private XMLLanguageService languageService;
+
+	@BeforeEach
+	public void setUp() throws IOException {
+		languageService = new XMLLanguageService();
+	}
+
+	@AfterEach
+	public void tearDown() throws InterruptedException, ExecutionException {
+		languageService.dispose();
+		languageService = null;
+	}
+
+	@Test
+	@Timeout(90000)
+	public void testManagedVersionHoverInDependencyChiild() throws IOException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-dependencyManagement-child.xml", languageService);
+		Hover hover = languageService.doHover(document, new Position(15, 25), new SharedSettings());
+		String value = ((MarkupContent) hover.getContents().getRight()).getValue();
+		System.out.println("testManagedVersionHoverInDependencyChiild: [" + value + "]");
+		assertNotNull(value);
+		assertTrue(value.contains("The managed version is"));
+		assertTrue(value.contains("2.22.2"));
+		assertTrue(value.contains("The artifact is managed in"));
+		assertTrue(value.contains("org.apache.maven.plugins:maven-surefire-plugin:2.22.2"));
```

Comment:
That's not correct. The artifact version is managed in pom-dependencyManagement-parent.

Reply:
Now a version from the parent pom and location of parent pom where the version is defined is shown in Hover.

If no version is specified in project it  shows a version and location of pom for an according artifact resolved in maven project.